src/components/ButtonDropdown.vue

```diff
@@ -10,7 +10,7 @@
       </span>
     </button>
     <div v-show="isOpen" class="button-dropdown-content" :style="dropdownStyle">
-      <slot></slot>
+      <slot v-if="isOpen"></slot>
```

Comment:
Ahhh I see, I think I got it backwards before -- so you _want_ to destroy/reset the date picker's state each time it is closed then re-opened, rather than preserve it, eh? Guessing this is to re-trigger the focus on mount? I think this is fine w/o parametrizing it, and overall re: the expensiveness to re-render, meh. 

But -- I think this is redundant with the enclosing div's `v-show="isOpen"` directive -- like ideally that one could just be modified to `v-if`, BUT poking around with that locally I just noticed that changing it accordingly leads to a new issue with the `.toolbar-item` styles which are incidentally working now, but are fragile and lead to a can of worms.

TLDR: IMO fine to either do it as-is, or just leave out the change to the button dropdown and I'll submit a PR separately which updates that along with the styling regression.